SofaKernel/modules/SofaLoader/MeshVTKLoader.cpp

```diff
@@ -898,7 +898,7 @@ bool LegacyVTKReader::readFile(const char* filename)
         }
         else if (!kw.empty())
         {
-            msg_error(this) << "WARNING: Unknown keyword " << kw ;
+            msg_warning(this) << "WARNING: Unknown keyword " << kw ;
```

Comment:
if you know when the new data has been introduce, maybe change the message to "vtk file version XX not handled"

Reply:
My 2cents,
You could also:
1) remove the "WARNING:" prefix
2) if the component is working properly then it make sense to have the message in either warning or info but not as an error.